v1/random.yaml

```diff
@@ -37,7 +37,7 @@ paths:
               status: 303
               headers:
                 cache_control: '{{options.random_cache_control}}'
-                location: '../../{request.params.format}/{title_from_mobileapps.body.title}'
+                location: '../../{request.params.format}/{title_from_mobileapps.body.items[0].title}'
```

Comment:
Sounds good to me. Thanks!

On Thu, Jul 14, 2016 at 11:05 AM, Marko Obrovac <notifications@github.com>
wrote:

> In v1/random.yaml
> <https://github.com/wikimedia/restbase/pull/636#discussion_r70843472>:
>
> > @@ -37,7 +37,7 @@ paths:
> >                status: 303
> >                headers:
> >                  cache_control: '{{options.random_cache_control}}'
> > -                location: '../../{request.params.format}/{title_from_mobileapps.body.title}'
> > +                location: '../../{request.params.format}/{title_from_mobileapps.body.items[0].title}'
>
> Even though I too find in superfluous, let's keep it as-is now for
> sanity's sake. Also, it needs to stay as-is if we want to get it out
> tomorrow.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wikimedia/restbase/pull/636/files/ba0d02e8c232a8d7845292bb5f98b4eecd7524ed..7dc87edc06644d1ea1bf67c595b44508b2236701#r70843472>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AByd28WOpAlyPRK0ncMg8K8jSbMvVe9rks5qVmxPgaJpZM4JIHPZ>
> .
>